system/core/pages.php

```diff
@@ -601,6 +601,19 @@ function wikify_link($node) {
 			$node->text('');
 	}
 
+	function figcaption($element) {
+		foreach ($element->findXPath(".//img[@longdesc]") as $img) {
+			figcaption_img($img);
+		}
+	}
+
+	function figcaption_img($img) {
```

Comment:
I would rename these functions to more descriptive names, maybe `add_captions_to_all_images` and `add_caption_to_image`?

Reply:
fixed!